src/Microsoft.Azure.ServiceBus/Amqp/AmqpConnectionHelper.cs

```diff
@@ -99,6 +99,23 @@ internal class AmqpConnectionHelper
             return tpSettings;
         }
 
+        public static TransportSettings CreateWebSocketTransportSettings(
+            string networkHost,
+            string hostName,
+            int port)
+        {
+            UriBuilder uriBuilder = new UriBuilder(WebSocketConstants.WebSocketSecureScheme, networkHost, port < 0 ? WebSocketConstants.WebSocketSecurePort : port, WebSocketConstants.WebSocketDefaultPath);
```

Comment:
@NatMarchand Fully understand. Reply to this question should address both. @nemakam ?